src/classes/ApexCollections.cls

```diff
@@ -0,0 +1,102 @@
+/**
+*  Description     :  Class for Practising Apex Collections
+*
+*  Created By      :  Mrinal Das
+*
+*  Created Date    :  06/24/2022
+*
+*  Revision Logs   :    V1.0 - Created - Mrinal Das
+*				   
+*
+**/
+
+public with sharing class ApexCollections {
+    
+    /**
+     * 
+     *   @description    :   practising of declaring a list, acessing elements from a list and adding value to One dimensional list
+     *
+     *   @args           :   
+     *
+     *   @return         :   Void
+     *
+     *   @revision Log   :   V1.1 - Created  - 06/24/2022 - Mrinal Das
+     *		          
+     * 
+     **/
+
+    public static void listInApex(){
+        //Create an empty list of String
+        List<String> my_list = new List<String>();
```

Comment:
you don't have to add _ you have to follow camel case
myList and your variable name is my_list this wrong